addons/l10n_sa/views/account_move_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="account_view_invoice_tree_inherit_l10n_sa" model="ir.ui.view">
+            <field name="name">account.view.invoice.tree.inherit</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_invoice_tree"/>
```

Comment:
If we really want it, you will need to override instead the 2 views `view_out_credit_note_tree` and `view_out_invoice_tree`.
So that bills don't have the field